packages/plugin-relay/src/global-types.ts

```diff
@@ -355,6 +356,19 @@ declare global {
       name?: string;
       edgesNullable?: EdgeNullability;
       nodeNullable?: NodeNullability;
+      edgesField?: Omit<
+        ObjectFieldOptions<
+          Types,
+          {},
+          ObjectRef<{}>,
+          Types['DefaultNodeNullability'],
+          {},
+          GlobalIDShape<Types> | string
+        >,
+        'args' | 'nullable' | 'resolve' | 'type'
+      > & {
+        nullable?: Types['DefaultNodeNullability'];
```

Comment:
I think the nullable for these needs to be removed here since we already have an existing option for that (same for node)

Reply:
technically this would have been better, if I had done it this way all along, but since the edgesNullable and nodeNullable options already exist, this will just add confusion